code.js

```diff
@@ -0,0 +1,121 @@
+const userPanel = document.querySelector('.userpanel');
+const canvas = document.getElementById('my-canvas');
+const modePicker = document.getElementById('modepicker');
+const linePicker = document.getElementById('linepicker');
+const colorPicker = document.getElementById('colorpicker');
+const buttonClear = document.querySelector('.button_clear');
+const buttonSave = document.querySelector('.button_save');
+const ctx = canvas.getContext('2d');
+
+let strokeStile = '#333333';
+console.log(userPanel.offsetHeight);
+canvas.width = window.innerWidth;
+canvas.height = window.innerHeight - userPanel.offsetHeight;
+
+ctx.strokeStyle = strokeStile;
+ctx.lineJoin = 'round';
+ctx.lineCap = 'round';
+ctx.lineWidth = 30;
+
+let isDrawing = false;
+let lastX = 0;
+let lastY = 0;
+let hue = 0;
+let direction = true;
+
+
+
+//Listeners
+canvas.addEventListener('mousemove', selectDrawingMode);
+canvas.addEventListener('mousedown', (e) => {
+    isDrawing = true;
+    [lastX, lastY] = [e.offsetX, e.offsetY];
+});
+canvas.addEventListener('mouseup', () => isDrawing = false);
+canvas.addEventListener('mouseout', () => isDrawing = false);
+
+buttonClear.addEventListener('click', clearCanvas);
+buttonSave.addEventListener('click', saveImage);
+
+
+
+function selectDrawingMode(e) {
+    if (modePicker.value === 'random') {
+        drawRandom(e);
+    } else {
+        drawAsArtist(e);
+    }
+
+}
+
+function drawAsArtist(e) {
+    if (!isDrawing) {
+        return;
+    }
+    ctx.beginPath();
+    ctx.strokeStyle = colorPicker.value;
+    ctx.lineWidth = linePicker.value;
+    ctx.moveTo(lastX, lastY);
+    ctx.lineTo(e.offsetX, e.offsetY);
+    ctx.stroke();
+    [lastX, lastY] = [e.offsetX, e.offsetY];
+}
+
+function drawRandom(e) {
+    if (!isDrawing) {
+        return;
+    }
```

Comment:
My opinion, if you have not complicated condition, it will be better write this in one row. For example 
If (!isDrawing) return;

Reply:
Now I do it for educational purposes :)))